ethereum-network-partition-test/main_test.go

```diff
@@ -74,7 +74,6 @@ const (
 
 	firstPartition  = "partition0"
 	secondPartition = "partition1"
```

Comment:
nit: Maybe we should align with the vocable we chose for Starlark, and change that to `subnetwork0` and `subnetwork1`